proprietary/design-tokens/src/component/utrecht/link-social.tokens.json

```diff
@@ -0,0 +1,15 @@
+{
+  "utrecht": {
+    "link-social": {
+      "background-color": { "value": "{utrecht.color.red.40}" },
+      "border-width": { "value": "2px" },
+      "border-color": { "value": "{utrecht.color.white}" },
+      "size": { "value": "40px" },
+      "distanced": { "value": "9.6px" },
+      "icon": {
+        "color": { "value": "{utrecht.color.white}" },
+        "size": { "value": "40px" }
```

Comment:
Do we really need 40px for icon size if we already have 40px for the link size? If not strictly needed, I suggest we start with fewer design tokens.